packages/ui/src/theme/css/component/fileUploader.scss

```diff
@@ -36,3 +37,48 @@
   font-size: var(--amplify-font-sizes-small);
   margin-block-start: var(--amplify-space-small);
 }
+
+.amplify-fileuploader__previewer {
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  max-height: 40rem;
+  max-width: 25rem;
+  overflow: hidden;
+}
+
+.amplify-fileuploader__previewer__body {
+  display: flex;
+  flex: 1;
+  overflow: auto;
+  flex-direction: column;
+  gap: var(--amplify-space-small);
+  padding: var(--amplify-space-medium);
+}
+
+.amplify-fileuploader--footer {
+  position: relative;
+  padding: var(--amplify-space-medium);
+  display: flex;
+  flex-direction: row;
+  border-top: var(--amplify-border-widths-small) solid
+    var(--amplify-colors-border-secondary);
+}
+
+.amplify-fileuploader--footer > .amplify-fileuploader-loader {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+.amplify-fileuploader--footer > *:first-child {
+  flex: 1;
+}
+
+.amplify-fileuploader-file {
+  .amplify-image {
+    object-fit: cover;
```

Comment:
The image will be clipped to fit if it's aspect ratio doesn't match, is that the intended behavior?

Reply:
I think it's fine, it's going to be used in the next PR with the file uploader file. We can change it though if needed.